### PR TITLE
[AMB-481] chore: Handle null `encryption_pubkey` field in non-banco contacts to avoid `NoCacheableError`

### PR DESCRIPTION
**Pull request by wthrajat**

### CODE REVIEW

File: src/api/contact/contact.resolver.ts [hunk 1]
```diff
@@ -108,7 +108,9 @@ export class WalletContactResolver {
   @ResolveField()
   async encryption_pubkey(@Parent() parent: WalletContactParent) {
     if (!parent.money_address) return null;
-    return this.isomorphicLnurl.getPubkey(parent.money_address);
+    const pubkey = await this.isomorphicLnurl.getPubkey(parent.money_address);
+    if (pubkey === null) return null;
```
Comment (wthrajat) [inline src/api/contact/contact.resolver.ts:112]: @bufo24 should we throw an error here?
```suggestion
    if (pubkey === null) {
      throw new Error(`Pubkey not found for money address: ${parent.money_address}`);
    }
```

Comment (bufo24) [reply src/api/contact/contact.resolver.ts:112]: we can leave this file as is.
`getPubkey` already returns either a string or null, and this field is nullable so this field resolver is all good

File: src/libs/lnurl/handlers/isomorphic.service.ts [hunk 2]
```diff
@@ -164,7 +164,6 @@ export class LnUrlIsomorphicService {
 
     if (isProd) {
       const result = moneyAddressType.safeParse(money_address);
-
```
Comment (bufo24) [inline src/libs/lnurl/handlers/isomorphic.service.ts:167]: NIT: unneeded change

File: src/api/contact/contact.resolver.ts (hunk 1 above)
Comment (wthrajat) [reply src/api/contact/contact.resolver.ts]: Right!

**Review by bufo24 — APPROVED** [review]

**Pull request merged**